### ISSUE TITLE
Display & Update Blob Metadata

### ISSUE THREAD

**Issue by sedouard** [azure-storage/xplat#244]
Add a new tab to the blob properties dialog for blob user-defined metadata modification.

**Comment by ammills01** [comment]
Yes, this would be excellent.  It would also be nice to allow us to set default metadata so we can choose things like TTL if we wanted the content to be refreshed more than once every 7 days.
